baseframe/forms/auto.py

```diff
@@ -17,25 +18,28 @@ class ConfirmDeleteForm(Form):
     cancel = SubmitField(__(u"Cancel"))
 
 
-def render_form(form, title, message='', formid='form', submit=__(u"Submit"), cancel_url=None, ajax=False):
+def render_form(form, title, message='', formid='form', submit=__(u"Submit"), cancel_url=None, ajax=False, with_chrome=True, error_template=False):
     multipart = False
     for field in form:
         if isinstance(field.widget, wtforms.widgets.FileInput):
             multipart = True
     if form.errors:
+        # TODO: Why is this 200?
         code = 200  # 400
```

Comment:
Because HTTP 400 breaks `$.ajax`. Instead of rendering the new response with errors, it was firing the client-side error handler, which (depending on the app's code) says the server couldn't be contacted.